src/stravalib/model.py

```diff
@@ -38,6 +38,7 @@
 from dateutil.parser import ParserError
 from pydantic import (
     AliasChoices,
+    AnyHttpUrl,
```

Comment:
We can remove this. i started to type URL's as this given it's the proper type for a url string (i think) for photos, etc. but strava_model doesn't use this at all.